VCEntities/VCEntities/verifiableCredential/VCClaims.swift

```diff
@@ -5,13 +5,19 @@
 
 import VCToken
 
+/**
+ * Contents of a Verifiable Credential Token.
+ *
+ * @see [Verifiable Credential Spec](https://www.w3.org/TR/vc-data-model/)
+ */
 public struct VCClaims: Claims {
-    public let jti: String
-    public let iss: String
-    public let sub: String
-    public let iat: Double
-    public let exp: Double
-    public let vc: VerifiableCredentialDescriptor
+    public let jti: String?
```

Comment:
Should we validate the VCs before saving them in issuance? if they're missing a required property they won't be valid.

Reply:
yes as we discussed, going to create an object validator